textattack/attack_recipes/iga_wang_2019.py

```diff
@@ -0,0 +1,55 @@
+from textattack.constraints.grammaticality.language_models import (
+    Google1BillionWordsLanguageModel,
+)
+from textattack.constraints.overlap import MaxWordsPerturbed
+from textattack.constraints.pre_transformation import (
+    RepeatModification,
+    StopwordModification,
+)
+from textattack.constraints.semantics import WordEmbeddingDistance
+from textattack.goal_functions import UntargetedClassification
+from textattack.search_methods import ImprovedGeneticAlgorithm
+from textattack.shared.attack import Attack
+from textattack.transformations import WordSwapEmbedding
+
+
+def IGAWang2019(model):
+    """
+        Xiaosen Wang, Hao Jin, Kun He (2019). 
+        
+        Natural Language Adversarial Attack and Defense inWord Level. 
+        
+        http://arxiv.org/abs/1909.06723 
+    """
+    #
+    # Swap words with their embedding nearest-neighbors.
+    #
+    # Embedding: Counter-fitted Paragram Embeddings.
+    #
+    # "[We] fix the hyperparameter values to S = 60, δ = 0.5, λ = 5, and N is unrestricted (50)."
```

Comment:
Can you move this comment to be above where you define `search_method`. Right now it looks like it's about `WordSwapEmbedding`

Reply:
Thanks for your comments, I will be revising the code in the next few days.